index.html

```diff
@@ -556,6 +556,30 @@ <h3 id="navigation-scope-security-considerations">
       </p>
       <section>
         <h3>
+          Privacy and security considerations
+        </h3>
+        <p>
+          When the <a>fullscreen</a> <a>display mode</a> is applied, it is
+          RECOMMENDED that the user agent still afford the end-user a means to
```

Comment:
it afford*s* but I am not sure afford is the right word here. Why not use 'allows'